scripts/testing/.gitignore

```diff
@@ -0,0 +1,4 @@
+nohup.out
+Vera.ttf
```

Comment:
This is needed to generate the graphs, correct? If so, shouldn't this be committed to the source repo?

Reply:
It is not needed, this is automatically downloaded once by `heatmap.py`.
I left this out, in case some day `heatmap.py` does something else or in a system where the font is already installed.